packages/lib/services/interop/InteropService_Importer_OneNote.postprocessHtml.test.ts

```
@@ -2,8 +2,42 @@ import '../../testing/test-utils';
+		const testImportDirectory = Setting.value('tempDir');
+		const onErrorCalls: string[] = [];
+
+		const importer = new InteropService_Importer_OneNote();
+		await importer.init(testImportDirectory, {
+			domParser: new DOMParser(),
+			xmlSerializer: new XMLSerializer(),
+			onError: (error: string|Error) => onErrorCalls.push(String(error)),
+		});
+
+		const importerWithInternals = importer as unknown as {
+			postprocessGeneratedHtmlInFolder_: (baseFolder: string)=> Promise<void>;
```

ESLint

[error] 'baseFolder' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)